docassemble/MOHUDEvictionProject/data/questions/court_information.yml

```
@@ -13,9 +13,46 @@ code: |
     else:
       trial_court.county = trial_court_county_selected
       trial_court.circuit = county_dictionary[trial_court.county]['circuit']
+  if len(county_dictionary[trial_court.county]["courts"]) == 1:
+    trial_court.location_name = None
+  else:
+    potential_courts = list()
+    for index in range(len(county_dictionary[trial_court.county]["courts"])):
+      potential_courts.append({index:county_dictionary[trial_court.county]["courts"][index]["location_name"]})
   trial_court.division = "Associate Division"
   trial_court.address.county = trial_court.county
 ---
+code: |
+  if len(county_dictionary[trial_court.county]["courts"]) == 1:
+    trial_court.address.address = county_dictionary[trial_court.county]["courts"][0]["address"]
+    trial_court.building_name = county_dictionary[trial_court.county]["courts"][0]["name"]
+    if "unit" in county_dictionary[trial_court.county]["courts"][0].keys():
```

```suggestion
    if "unit" in county_dictionary[trial_court.county]["courts"][0]:
```

`in` automatically implies searching `keys` fwiw